src/setuptools_scm/version.py

```diff
@@ -147,8 +147,12 @@ def _parse_tag(tag, preformatted):
     return tag
 
 
-def meta(tag, distance=None, dirty=False, node=None, preformatted=False, **kw):
-    parsed_version = _parse_tag(tag, preformatted)
+def meta(tag, distance=None, dirty=False, node=None, preformatted=False,
```

Comment:
actually, its enough to pass preformatted, which does the same in parse_tag, jsut see above

man i really should have added that flag back then

Reply:
Indeed, much simpler this way !